packages/contracts/contracts/discovery/pools/PeoplesPool.sol

```diff
@@ -2,5 +2,15 @@
 pragma solidity =0.8.12;
 
 import {Pool} from "./Pool.sol";
+import {IBatch} from "../interfaces/IBatch.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
-contract PeoplesPool is Pool {}
+contract PeoplesPool is Pool {
+
+  function invest(address _user, address _project, address _batch, uint256 _amount) onlyController {
+    require(IBatch(_batch).canInvestInPeoplesPool(_user, _project));
```

Comment:
This is on the controller now. Should probably be called internally by the controller, since this invest can only be called by the controller anyway, no?